packages/calypso-e2e/src/lib/blocks/block-flows/opentable.ts

```diff
@@ -48,7 +48,7 @@ export class OpenTableFlow implements BlockFlow {
 		 * Due to API inconsistency of OpenTable, we need to try a few times to get the correct restaurant.
 		 * https://github.com/Automattic/wp-calypso/issues/92836
 		 */
-		for ( let i = 0; i < 5; i++ ) {
+		for ( let i = 0; i < 10; i++ ) {
```

Comment:
This was still failing: https://github.com/Automattic/wp-calypso/issues/92836#issuecomment-2248225447